libs/api/metadata-converter/src/lib/iso19139/iso19139.converter.ts

```diff
@@ -398,10 +398,11 @@ export class Iso19139Converter extends BaseConverter<string> {
         this.writers['spatialExtents'](record, rootEl)
       fieldChanged('lineage') && this.writers['lineage'](record, rootEl)
     }
-
     fieldChanged('otherLanguages') &&
       this.writers['otherLanguages'](record, rootEl)
 
+    fieldChanged('translations') && this.writers['translations'](record, rootEl)
```

Comment:
see l. 163 https://github.com/geonetwork/geonetwork-ui/pull/1250/files#diff-fb106e4a7ec88ef749c3d7f2b3a5c0dc559b7ddb9792f134b62cb0be82d4a3f3R163 the dedicated write does nothing.